### DISCUSSION TITLE
Link to the exercise and overall time on header

### DISCUSSION THREAD

**Discussion by rauldub** [jlai/fitness-dashboard#45] [Ideas]
Hello, 

After clicking on particular exercise, like run on "Activity logs", would be great to have overall time also somewhere on the top. 

I'd love to have a possibility to copy/store link to particular exercise as well.

**Comment by jlai** [comment]
I've added the exercise duration to the header. Also, when you view an activity from the history page, it should now include the activity log id in the url which you can copy/paste/bookmark to return to that activity.

**Reply by rauldub** [reply]
Wow, it was fast! Fitbit support would react probably.. never ;) Thank you! 

edit: id's correspondent original Fitbit page, what is awesome! Double thank you then!
